_sass/card.scss

```diff
@@ -6,6 +6,20 @@
 	border-radius: 5px; /* 5px rounded corners */
 	display: inline-block;
 	margin: 10px;
+
+	& .elastic-fai {
+		font-size: $type-size-1;
```

Comment:
```suggestion
		font-size: $type-size-1;
		margin-bottom: 0.5em;
```
I'd add a margin here, too, otherwise they are too close to the buttons. I checked, it's also okay for the desktop

![image](https://user-images.githubusercontent.com/9960249/84568435-c6717e80-ad7f-11ea-842a-c7e51c4ba0b3.png)

Reply:
Yes, thanks. A margin is great here :) I was a bit afraid because of the transition between the different sizes.